src/include/baselib/data/eh/ServerErrorHelpers.h

```diff
@@ -25,6 +25,8 @@
 #include <baselib/core/ObjModel.h>
 #include <baselib/core/BaseIncludes.h>
 
+#include <baselib/messaging/BrokerErrorCodes.h>
```

Comment:
The data library should not have such dependency on the messaging library as it is much lower in the stack